core/main/src/Options/Classes/Theme/Theme.ts

```diff
@@ -0,0 +1,27 @@
+import type {ITheme} from "../../Interfaces/Theme/ITheme"
+import type { RecursivePartial } from "../../../Types";
+import {IOptionLoader} from "../../Interfaces/IOptionLoader"
+export class Theme implements ITheme, IOptionLoader<ITheme>{
+   
+    public name: string;
+    public default: boolean;
+    
+    constructor() {
+        this.name = "dark";
+        this.default = false;
+    }
+    public load(data?: RecursivePartial<ITheme>): void {
+        if (data === undefined) {
+            return;
+        }
+
+        if (data.name !== undefined) {
+            this.name = data.name;
+        }
+
+        if (data.default !== undefined) {
+            this.default = data.default;
+        }
+    }
```

Comment:
This is needed the load of the `options` property described above like the file `Trail.ts` does for `particles` property, located under Options Interactivity Modes